src/services/analytics/gtm.ts

```diff
@@ -56,10 +76,14 @@ export const gtmInit = (): void => {
     return
   }
 
+  const pagePath = getAnonymizedPathname()
+
   TagManager.initialize({
     gtmId: GOOGLE_TAG_MANAGER_ID,
     ...GTM_ENVIRONMENT,
     dataLayer: {
+      pageLocation: `${location.origin}${pagePath}`,
```

Comment:
In my tests, `router.pathname` doesn't include our rewrites. We could pass it as an argument to `gtmInit` and then not need anonymise the `pathname`.

Reply:
Good idea! It works nicely although we have to exclude it from the dependency array but I think it should be fine since it is only needed on the initial load.